### PR TITLE
MicroProfile 5.0 TCK results

### PR DESCRIPTION
**Pull request by Azquelt**

### CODE REVIEW

File: microprofile/5.0/health/21.0.0.12-beta-TCKResults.adoc [hunk 1]
```diff
@@ -0,0 +1,188 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Health 4.0.
+
+== Open Liberty 21.0.0.12-beta - MicroProfile Health 4.0 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/beta/openliberty-runtime/21.0.0.12-beta/openliberty-runtime-21.0.0.8-beta.zip[Open Liberty 21.0.0.12-beta]
```
Comment (Azquelt) [inline microprofile/5.0/health/21.0.0.12-beta-TCKResults.adoc]: Need to fix runtime version in link

File: microprofile/5.0/openapi/21.0.0.12-beta-TCKResults.adoc [hunk 2]
```diff
@@ -0,0 +1,607 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Open API 3.0.
+
+== Open Liberty 21.0.0.12-beta - MicroProfile Open API 3.0 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/beta/openliberty-runtime/21.0.0.12-beta/openliberty-runtime-21.0.0.8-beta.zip[Open Liberty 21.0.0.12-beta]
```
Comment (Azquelt) [inline microprofile/5.0/openapi/21.0.0.12-beta-TCKResults.adoc]: Fix this runtime version too

File: microprofile/5.0/health/21.0.0.12-beta-TCKResults.adoc (hunk 1 above)
Comment (Yas-Franco) [reply microprofile/5.0/health/21.0.0.12-beta-TCKResults.adoc]: Done!

File: microprofile/5.0/openapi/21.0.0.12-beta-TCKResults.adoc (hunk 2 above)
Comment (Yas-Franco) [reply microprofile/5.0/openapi/21.0.0.12-beta-TCKResults.adoc]: Done!

**Review by tevans78 — DISMISSED** [review]

**Review by tevans78 — APPROVED** [review]

**Pull request merged**